core/iwasm/interpreter/wasm_loader.c

```diff
@@ -5711,6 +5711,7 @@ orcjit_thread_callback(void *arg)
 static void
 orcjit_stop_compile_threads(WASMModule *module)
 {
+#if WASM_ENABLE_LAZY_JIT != 0
```

Comment:
To avoid compilation warnings?

Reply:
it was a fix for a CI failure observed in this PR: https://github.com/bytecodealliance/wasm-micro-runtime/pull/4213#issuecomment-2857879014
i submitted a separate PR for this part: https://github.com/bytecodealliance/wasm-micro-runtime/pull/4252